test/unit/PollCreator.js

```diff
@@ -0,0 +1,70 @@
+import truffleAssert from "truffle-assertions"
+
+import Fixture from "./helpers/Fixture"
+import expectRevertWithReason from "../helpers/expectFail"
+import {functionSig} from "../../utils/helpers"
+
+const PollCreator = artifacts.require("PollCreator")
+const GenericMock = artifacts.require("GenericMock")
+
+const QUORUM = 20
+const THRESHOLD = 50
+const POLL_PERIOD = 10 * 5760
+
+contract("PollCreator", accounts => {
+    let fixture
+    let token
+    let pollCreator
+
+    before(async () => {
+        fixture = new Fixture(web3)
+        token = await GenericMock.new("LivepeerToken")
```

Comment:
The `GenericMock` constructor shouldn't accept any arguments (but I guess JS doesn't complain about that and they're just ignored)